functions/package.json

```diff
@@ -8,23 +8,33 @@
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
-    "setup-firebase": "node setupProject.js"
+    "setup-firebase": "node setupProject.js",
+    "test": "mocha -r ts-node/register --recursive 'test/**/*.test.ts' --timeout 10000"
```

Comment:
Why do we need a timeout?

Reply:
That's because mocha has a 2000 milliseconds delay by default. But some tests sometimes take more than 2 seconds to execute, and if we don't add timeout then it fails. Of course it's not the happiest solution :/